apps/condo/domains/notification/adapters/redStoreAdapter.js

```diff
@@ -0,0 +1,145 @@
+const { isEmpty, isNull, get } = require('lodash')
+
+const conf = require('@open-condo/config')
+const { getLogger } = require('@open-condo/keystone/logging')
+
+const { RedStoreNotificationSender } = require('@condo/domains/notification/adapters/redStore/redStoreNotificationSender')
+const {
+    REDSTORE_CONFIG_ENV,
+    PUSH_TYPE_DEFAULT,
+    APPS_WITH_DISABLED_NOTIFICATIONS_ENV,
+} = require('@condo/domains/notification/constants/constants')
+
+const REDSTORE_CONFIG = conf[REDSTORE_CONFIG_ENV] ? JSON.parse(conf[REDSTORE_CONFIG_ENV]) : null
+const APPS_WITH_DISABLED_NOTIFICATIONS = conf[APPS_WITH_DISABLED_NOTIFICATIONS_ENV] ? JSON.parse(conf[APPS_WITH_DISABLED_NOTIFICATIONS_ENV]) : []
+const DEFAULT_PUSH_SETTINGS = {}
+
+const logger = getLogger('redStoreAdapter')
+
+/**
+ * RedStore rejects push if any of data fields is not a string, so we should convert all non-string fields to strings
+ */
+function prepareData (data = {}, token) {
+    const result = { token }
+
+    for (const key in data) {
+        if (data.hasOwnProperty(key)) result[key] = String(data[key])
+    }
+
+    return result
+}
+/**
+ * Send push notification to pushToken via app, configured by REDSTORE_CONFIG in .helm (.env)
+ * Attention! Notifications could only be sent to devices, connected via same PROJECT_ID.
+ * Attempts to send push notifications to devices, connected through different projects will fail.
+ */
+class RedStoreAdapter {
+    _config = null
+
+    constructor (config = REDSTORE_CONFIG) {
+        try {
+            if (isEmpty(config)) throw new Error(`Valid ${REDSTORE_CONFIG_ENV} should be provided within .helm (.env)`)
```

Comment:
this is something odd - you are throwing an exception catch such exception in the next line of code. Just print a log in such case